code/datums/gamemode/factions/bloodcult/bloodcult_buildings.dm

```diff
@@ -560,7 +560,9 @@
 				var/datum/faction/bloodcult/cult = find_active_faction_by_type(/datum/faction/bloodcult)
 				if (!cult)
 					return
-				var/dat = {"<body style="color:#FFFFFF" bgcolor="#110000"><ul>"}
+				var/dat = {"<body style="color:#FFFFFF" bgcolor="#110000">"}
+				dat += "<b>We are currently allowed up to [cult.cultist_cap] members.</b>"
```

Comment:
You would think this would sound a little more culty. Like, I dunno.

`The Blood Harvest can sustain X more reapers.`
or even
`The Geometer will bless the conversion of X more servants.`

Reply:
I'll change the wording a bit